Readability.js

```diff
@@ -896,6 +896,14 @@ Readability.prototype = {
       let shouldRemoveTitleHeader = true;
 
       while (node) {
+
+        if(node.tagName === "HTML") {
+          var articleLang = node.getAttribute("lang");
+          if(articleLang) {
+            this._articleLang = articleLang;
```

Comment:
I don't think it's necessary for us to have this check here, actually. We could just get the "lang" attribute  and directly assign it.

```
this._articleLang = node.getAttribute("lang");
```

Reply:
if (node.tagName === "HTML") {
          var articleLang = node.getAttribute("lang");
          this._articleLang = articleLang;
        }
I just want to make sure I understand, like this? And then would I run "npm run lint && npm test" in readability not mozilla-unified, correct?

Reply:
Yeah! Although I think we can (and this is more of a nitpick)  reduce this by one more line:

```
    if (node.tagName === "HTML") {
      this._articleLang = node.getAttribute("lang");
    }
```

And you're correct. You should run `npm run lint && npm test` from the readability repo.